src/config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "version": "1.0",
+  "issuer": "http://localhost:8080/auth/realms/appuio",
+  "clientId": "appuio-cloud-portal"
```

Comment:
Please note that we are about to engineer a local dev environment complete with Kubernetes-in-Docker and an already-running Keycloak in https://github.com/appuio/control-api/pull/9.
That way, you don't need to run Keycloak locally and waste precious CPU/RAM on local machine with a big Java app. Also, you'll need custom APPUiO-specific Keycloak mappers at some point.
But for the moment, let's keep the docker-compose and see how the other setup is coming up.

Reply:
Sounds cool, I am looking forward to test it ;)